packages/addons/addon-depends/ffmpegx/package.mk

```diff
@@ -7,7 +7,7 @@ PKG_SHA256="06b10a183ce5371f915c6bb15b7b1fffbe046e8275099c96affc29e17645d909"
 PKG_LICENSE="LGPLv2.1+"
 PKG_SITE="https://ffmpeg.org"
 PKG_URL="https://ffmpeg.org/releases/ffmpeg-${PKG_VERSION}.tar.xz"
-PKG_DEPENDS_TARGET="toolchain aom bzip2 gnutls lame libvorbis opus x264 x265 zlib"
```

Comment:
x265 is only used for x86_64 and the dependency already added in line 18.

Does x265 build on any platform?

Reply:
Looks to be supported on all ARM:

- https://x265.readthedocs.io/en/master/releasenotes.html#version-3-4
- Version 3.4
  - Faster ARM64 encodes enabled by ASM contributions from Huawei. The speed-up over no-asm version for 1080p encodes @ medium preset is ~15% in a 16 core H/W. 
- https://x265.readthedocs.io/en/master/releasenotes.html#version-2-0 
- Version 2.0
  - asm: ARM assembly is now enabled by default, native or cross compiled builds supported on armv6 and later systems.

It is currently disabled from build in the package.mk file, but with the following changes it compiles on both ARMv7 / ARMv8.
- `PROJECT=ARM ARCH=arm DEVICE=ARMv7 s/build x265`
- `PROJECT=ARM ARCH=arm DEVICE=ARMv8 s/build x265`

```
--- a/packages/addons/addon-depends/ffmpegx-depends/x265/package.mk
+++ b/packages/addons/addon-depends/ffmpegx-depends/x265/package.mk
@@ -4,12 +4,12 @@
 PKG_NAME="x265"
 PKG_VERSION="3.5"
 PKG_SHA256="7ebc5d2de6ce5dfefb434e422e59a0c4715fe939c784ac2f3d41af5775adc706"
-PKG_ARCH="x86_64"
 PKG_LICENSE="GPL"
 PKG_SITE="https://www.videolan.org/developers/x265.html"
 PKG_URL="https://bitbucket.org/multicoreware/x265_git/get/${PKG_VERSION}.tar.bz2"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="x265 is a H.265/HEVC video encoder application library"
+PKG_BUILD_FLAGS="+pic"
 PKG_TOOLCHAIN="make"

 pre_configure_target() {
```